vnext/codegen/react/components/rnwcore/Props.cpp

```diff
@@ -0,0 +1,196 @@
+    mode(convertRawProp(rawProps, "mode", sourceProps.mode, {DatePickerMode::Date})),
+    timeZoneOffsetInMinutes(convertRawProp(rawProps, "timeZoneOffsetInMinutes", sourceProps.timeZoneOffsetInMinutes, {0.0}))
+      {}
+AndroidDrawerLayoutProps::AndroidDrawerLayoutProps(
+    const AndroidDrawerLayoutProps &sourceProps,
+    const RawProps &rawProps): ViewProps(sourceProps, rawProps),
+
+    keyboardDismissMode(convertRawProp(rawProps, "keyboardDismissMode", sourceProps.keyboardDismissMode, {AndroidDrawerLayoutKeyboardDismissMode::None})),
+    drawerBackgroundColor(convertRawProp(rawProps, "drawerBackgroundColor", sourceProps.drawerBackgroundColor, {})),
+    drawerPosition(convertRawProp(rawProps, "drawerPosition", sourceProps.drawerPosition, {AndroidDrawerLayoutDrawerPosition::Left})),
+    drawerWidth(convertRawProp(rawProps, "drawerWidth", sourceProps.drawerWidth, {})),
+    drawerLockMode(convertRawProp(rawProps, "drawerLockMode", sourceProps.drawerLockMode, {AndroidDrawerLayoutDrawerLockMode::Unlocked})),
+    statusBarBackgroundColor(convertRawProp(rawProps, "statusBarBackgroundColor", sourceProps.statusBarBackgroundColor, {}))
+      {}
+RCTMaskedViewProps::RCTMaskedViewProps(
+    const RCTMaskedViewProps &sourceProps,
+    const RawProps &rawProps): ViewProps(sourceProps, rawProps)
+
+    
+      {}
+AndroidProgressBarProps::AndroidProgressBarProps(
+    const AndroidProgressBarProps &sourceProps,
+    const RawProps &rawProps): ViewProps(sourceProps, rawProps),
+
+    styleAttr(convertRawProp(rawProps, "styleAttr", sourceProps.styleAttr, {})),
+    typeAttr(convertRawProp(rawProps, "typeAttr", sourceProps.typeAttr, {})),
+    indeterminate(convertRawProp(rawProps, "indeterminate", sourceProps.indeterminate, {false})),
+    progress(convertRawProp(rawProps, "progress", sourceProps.progress, {0.0})),
+    animating(convertRawProp(rawProps, "animating", sourceProps.animating, {true})),
+    color(convertRawProp(rawProps, "color", sourceProps.color, {})),
+    testID(convertRawProp(rawProps, "testID", sourceProps.testID, {""}))
+      {}
+RCTProgressViewProps::RCTProgressViewProps(
+    const RCTProgressViewProps &sourceProps,
+    const RawProps &rawProps): ViewProps(sourceProps, rawProps),
+
+    progressViewStyle(convertRawProp(rawProps, "progressViewStyle", sourceProps.progressViewStyle, {RCTProgressViewProgressViewStyle::Default})),
+    progress(convertRawProp(rawProps, "progress", sourceProps.progress, {0.0})),
+    progressTintColor(convertRawProp(rawProps, "progressTintColor", sourceProps.progressTintColor, {})),
+    trackTintColor(convertRawProp(rawProps, "trackTintColor", sourceProps.trackTintColor, {})),
+    progressImage(convertRawProp(rawProps, "progressImage", sourceProps.progressImage, {})),
+    trackImage(convertRawProp(rawProps, "trackImage", sourceProps.trackImage, {}))
+      {}
+AndroidSwipeRefreshLayoutProps::AndroidSwipeRefreshLayoutProps(
```

Comment:
is this right? should we have android properties in RNW? is there something that can be queried to filter out platform-specific apis?

Reply:
There is a property on the original definition of the component.  Ex: RCTInputAccessoryView is marked as not being supported in android.  Using that would be annoying though, since we'd need to update / fork core in order to mark things as being / not being supported in windows.
https://github.com/facebook/react-native/blob/b2dbde36b8f9cb69b1a1f9061c7f19c895d0c317/Libraries/Components/TextInput/RCTInputAccessoryViewNativeComponent.js#L25

I'm assuming that all these extra props classes etc will be stripped from the final ship dll since they are not referenced anywhere.